readthedocs/starterguides/userguides/fileshare_userguide.md

```diff
@@ -0,0 +1,106 @@
+## FileShare Introduction
+Fileshare in opensds facilitates the user to create fileshare, set access permission for that fileshare and user can also create snapshot. Opensds provides both CLI and Dashboard to operate or play with fileshare feature.
+
+### Install fileshare feature with opensds
+Follow the wiki link to install opensds https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible
+
+Make sure that backend nfs is enable to use fileshare feature 
+###### NFS
+If `nfs` is chosen as storage backend, modify `group_vars/osdsdock.yml`, native nfs driver dependent on lvm:
+```yaml
+enabled_backends: nfs
+```
+
+## Set the environment to start use of fileshare feature
+
+### OpenSDS CLI
+Firstly configure opensds CLI tool:
+```bash
+sudo cp /opt/opensds-hotpot-linux-amd64/bin/osdsctl /usr/local/bin/
+
+export OPENSDS_ENDPOINT=http://{your_real_host_ip}:50040
+export OPENSDS_AUTH_STRATEGY=keystone
+export OS_AUTH_URL=http://{your_real_host_ip}/identity
+export OS_USERNAME=admin
+export OS_PASSWORD=opensds@123
+export OS_TENANT_NAME=admin
+export OS_PROJECT_NAME=admin
+export OS_USER_DOMAIN_ID=default
+
+osdsctl pool list # Check if the pool resource is available
+```
+
+#### Profile creation for fileshare
+* Create profile for fileshare
+
+```yaml
+osdsctl profile create '{"name":"test_profile", "description":"test policy for fileshare", "storageType":"file"}'
```

Comment:
I think this command is duplicate in this doc, please have a check.

Reply:
Yes i removed duplication.